docs/run-keep-ecdsa.adoc

```diff
@@ -260,6 +282,47 @@ reference].
 |No
 |===
 
+[%header,cols=4*]
+|===
+|`Extensions.TBTC`
+|Description
+|Default
+|Required
+
+|`LiquidationRecoveryTimeout`
+|The amount of time your client will try to communicate with the other signers to recover the underlying bitcoin from a deposit after it has been liquidated.
+|"48h"
+|Yes
```

Comment:
It's not required, right? Please correct me if I'm wrong but the only required property for the TBTC extension is `BeneficiaryAddress`, as all the others have their default values.